deployment/src/test/java/io/quarkiverse/quinoa/test/QuinoaCIConfigTest.java

```diff
@@ -21,14 +20,17 @@ public class QuinoaCIConfigTest {
             .ci(null)
             .toQuarkusUnitTest()
             .overrideConfigKey("quarkus.quinoa.ci", "true")
+            .assertException(e -> {
+                assertThat(e)
+                        .hasMessage("Error in Quinoa while running package manager ci command: yarn.cmd install --immutable");
+            })
             .assertLogRecords(l -> assertThat(l)
-                    .anyMatch(s -> s.getMessage().equals("Running Quinoa package manager ci command: %s") &&
-                            s.getParameters()[0].equals(
-                                    systemBinary(PackageManagerType.YARN.getBinary()) + " install --frozen-lockfile")));
+                    .anyMatch(s -> s.getMessage().contains(
+                            "The lockfile would have been modified by this install, which is explicitly forbidden.")));
```

Comment:
I think this is the real issue.  When I run this test locally this happens and it says the lock file has changed.  So now this test passes locally but now fails on the CI server.